src/js/base/module/Codeview.js

```diff
@@ -50,6 +50,34 @@ export default class CodeView {
     this.context.triggerEvent('codeview.toggled');
   }
 
+  /**
+   * purify code view
+   * @param value
+   * @returns {*}
+   */
+  purify(value) {
+    if (this.options.codeviewFilter) {
+      // filter code view regex
+      value = value.replace(this.options.codeviewRegex, '');
+      // allow specific iframe tag
+      var whitelist = this.options.codeviewIframeWhitelistSrc.concat(this.options.codeviewIframeWhitelistSrcOrigin);
+      value = value.replace(/(<iframe.*?>.*?<\/iframe>)/gi, function(tag) {
+        // remove src duplication
+        if (/<.+src(?==?('|"|\s)?)[\s\S]+src(?=('|"|\s)?)[^>]*?>/i.test(tag)) {
+          return '';
+        }
+        for (var i in whitelist) {
```

Comment:
We can use ES6 syntax to iterating over an Array.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Iterating_over_an_Array